src/오픈채팅방.js

```diff
@@ -0,0 +1,34 @@
+function solution(records) {
+  const answer = [];
+  const store = {};
+  const log = {};
+
+  for (let i = 0; i < records.length; i++) {
+    const record = records[i].split(" ");
+    const command = record[0];
+    const id = record[1];
+    const nickname = record[2];
```

Comment:
```suggestion
    const [command, id, nickname] = records[i].split(" ");
```
이렇게 구조 분해 할당을 이용해서도 가능할 꺼 같습니다!

Reply:
넵, 조금 리팩토링하고 올렸어야 했는데, 준형님 풀이보고 배웠습니다! 감사합니다!